utilities/RedcapParser.js

```diff
@@ -116,6 +127,8 @@ function REDCapConverter(redcap_json) {
           rawContent[key].field_type = 'range-info';
         } else if (rawContent[key].field_annotation.includes('info-type')) {
           rawContent[key].field_type = 'info';
+        } else if (rawContent[key].field_note.includes('info-type')) {
```

Comment:
Is this in field note or field annotation like others?

Reply:
Ah yes there were some questionnaires that had it in field_note as well, so I just added it.